parcels/codegenerator.py

```diff
@@ -165,7 +165,8 @@ def __getattr__(self, attr):
 class ErrorCodeNode(IntrinsicNode):
     symbol_map = {'Success': 'SUCCESS', 'Evaluate': 'EVALUATE', 'Repeat': 'REPEAT', 'Delete': 'DELETE',
                   'Error': 'ERROR', 'ErrorInterpolation': 'ERROR_INTERPOLATION',
-                  'ErrorOutOfBounds': 'ERROR_OUT_OF_BOUNDS', 'ErrorThroughSurface': 'ERROR_THROUGH_SURFACE'}
+                  'ErrorOutOfBounds': 'ERROR_OUT_OF_BOUNDS', 'ErrorThroughSurface': 'ERROR_THROUGH_SURFACE',
+                  'StopExecution': 'STOP_EXECUTION'}
```

Comment:
this may not be the best (though it is the easiest) place to add that error. I think it would be better to have all the general, pipeline-related flags before 'Error' - even though it means shifting the error codes. I though understand that 'Success' is necessarily at position 0 because that is the standard return code of C for successful executions. Still, I would semantically not just append this flag at the end.